packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -283,7 +283,6 @@ const TradeParamsMobile = observer(
                 {isVisible('duration') ? (
                     <div data-header-content={getHeaderContent('duration')}>
                         <DurationMobile
-                            // @ts-expect-error: TODO: check if TS error is gone after <DurationMobile /> is migrated to TS
```

Comment:
it looks like DurationMobile is migrated to TS in your PR :)

Reply:
Yes, it was among files for TS migration in my card, so I checked places, were `// @ts-expect-error` was used and did a small cleanup